spec/requests/api/v2/people_controller_index_spec.rb

```diff
@@ -0,0 +1,213 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V2::PeopleController do
+  let(:supplier) { create(:supplier) }
+  let!(:application) { create(:application, owner_id: supplier.id) }
+  let!(:access_token) { create(:access_token, application: application).token }
+  let(:headers) { { 'CONTENT_TYPE': content_type }.merge('Authorization' => "Bearer #{access_token}") }
+  let(:response_json) { JSON.parse(response.body) }
+  let(:content_type) { ApiController::CONTENT_TYPE }
+
+  describe 'GET /people' do
+    let(:schema) { load_yaml_schema('get_people_responses.yaml', version: 'v2') }
+    let!(:people) { create_list :person, 2 }
+    let(:params) { {} }
+
+    context 'when successful' do
+      context 'when there are no params' do
+        before { get '/api/v2/people', params: params, headers: headers }
```

Comment:
Should we include the `:skip_before` option, here :trollface:

Reply:
😱